[2024-03-15 14:12:08,224: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-03-15 14:12:08,226: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-15 14:12:08,227: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-15 14:12:08,227: INFO: common: created directory at: artifacts]
[2024-03-15 14:12:08,227: INFO: common: created directory at: artifacts/data_ingestion]
[2024-03-15 14:12:08,227: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-03-15 14:12:30,636: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-03-15 14:12:31,268: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-03-15 14:30:22,963: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-15 14:30:22,966: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-15 14:30:22,967: INFO: common: created directory at: artifacts]
[2024-03-15 14:36:11,517: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-15 14:36:11,518: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-15 14:36:11,519: INFO: common: created directory at: artifacts]
[2024-03-15 14:38:00,939: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-15 14:38:00,942: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-15 14:38:00,942: INFO: common: created directory at: artifacts]
[2024-03-15 14:39:10,291: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-15 14:39:10,293: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-15 14:39:10,294: INFO: common: created directory at: artifacts]
[2024-03-15 14:39:56,039: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-15 14:39:56,041: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-15 14:39:56,043: INFO: common: created directory at: artifacts]
[2024-03-15 14:39:56,043: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-03-15 14:40:05,858: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-03-15 14:50:29,933: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-03-15 14:50:29,933: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-15 14:50:29,933: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-15 14:50:29,933: INFO: common: created directory at: artifacts]
[2024-03-15 14:50:29,933: INFO: common: created directory at: artifacts/data_ingestion]
[2024-03-15 14:50:29,933: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-03-15 14:50:44,758: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-03-15 14:50:45,162: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-03-15 14:50:45,163: INFO: main: *******************]
[2024-03-15 14:50:45,163: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-03-15 14:50:45,164: ERROR: main: name 'PrepareBaseModelTrainingPipeline' is not defined]
Traceback (most recent call last):
  File "D:\Projects\Kidney-Disease-Classification-MLFlow\main.py", line 21, in <module>
    prepare_base_model = PrepareBaseModelTrainingPipeline()
NameError: name 'PrepareBaseModelTrainingPipeline' is not defined
[2024-03-15 14:51:22,143: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-03-15 14:51:22,154: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-15 14:51:22,154: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-15 14:51:22,154: INFO: common: created directory at: artifacts]
[2024-03-15 14:51:22,154: INFO: common: created directory at: artifacts/data_ingestion]
[2024-03-15 14:51:22,154: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-03-15 14:51:35,378: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-03-15 14:51:35,671: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-03-15 14:51:35,671: INFO: main: *******************]
[2024-03-15 14:51:35,671: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-03-15 14:51:35,679: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-15 14:51:35,679: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-15 14:51:35,679: INFO: common: created directory at: artifacts]
[2024-03-15 14:51:35,679: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-03-15 14:51:35,986: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-03-15 14:51:36,221: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-03-15 15:09:47,813: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-15 15:09:47,816: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-15 15:09:47,817: INFO: common: created directory at: artifacts]
[2024-03-15 15:09:47,818: INFO: common: created directory at: artifacts\training]
[2024-03-15 15:10:31,399: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-15 15:10:31,401: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-15 15:10:31,401: INFO: common: created directory at: artifacts]
[2024-03-15 15:10:31,402: INFO: common: created directory at: artifacts\training]
[2024-03-15 15:18:00,129: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-15 15:18:00,129: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-15 15:18:00,129: INFO: common: created directory at: artifacts]
[2024-03-15 15:18:00,134: INFO: common: created directory at: artifacts\training]
[2024-03-15 15:20:50,375: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-15 15:20:50,383: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-15 15:20:50,383: INFO: common: created directory at: artifacts]
[2024-03-15 15:20:50,383: INFO: common: created directory at: artifacts\training]
[2024-03-15 15:28:52,154: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-03-15 15:28:52,154: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-15 15:28:52,154: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-15 15:28:52,154: INFO: common: created directory at: artifacts]
[2024-03-15 15:28:52,154: INFO: common: created directory at: artifacts/data_ingestion]
[2024-03-15 15:28:52,154: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-03-15 15:29:07,704: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-03-15 15:29:07,706: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-15 15:29:07,707: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-15 15:29:07,707: INFO: common: created directory at: artifacts]
[2024-03-15 15:29:07,707: INFO: common: created directory at: artifacts/data_ingestion]
[2024-03-15 15:29:07,707: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-03-15 15:29:56,646: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-03-15 15:29:57,180: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-03-15 15:29:57,181: INFO: main: *******************]
[2024-03-15 15:29:57,181: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-03-15 15:29:57,182: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-15 15:29:57,184: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-15 15:29:57,185: INFO: common: created directory at: artifacts]
[2024-03-15 15:29:57,185: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-03-15 15:29:57,744: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-03-15 15:29:58,086: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-03-15 15:29:58,087: INFO: main: *******************]
[2024-03-15 15:29:58,087: INFO: main: >>>>>> stage Training started <<<<<<]
[2024-03-15 15:29:58,088: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-15 15:29:58,089: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-15 15:29:58,090: INFO: common: created directory at: artifacts]
[2024-03-15 15:29:58,091: INFO: common: created directory at: artifacts\training]
[2024-03-15 15:33:47,515: INFO: main: >>>>>> stage Training completed <<<<<<

x==========x]
[2024-03-15 15:40:57,161: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-15 15:40:57,163: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-15 15:40:57,164: INFO: common: created directory at: artifacts]
[2024-03-15 15:47:36,328: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-15 15:47:36,328: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-15 15:47:36,328: INFO: common: created directory at: artifacts]
[2024-03-15 15:47:44,286: INFO: common: json file saved at: scores.json]
[2024-03-15 15:47:47,061: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-03-15 15:47:47,425: INFO: builder_impl: Assets written to: C:\Users\svaru\AppData\Local\Temp\tmpvs5uhbnu\model\data\model\assets]
[2024-03-15 15:59:44,172: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-15 15:59:44,176: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-15 15:59:44,177: INFO: common: created directory at: artifacts]
[2024-03-15 15:59:52,951: INFO: common: json file saved at: scores.json]
[2024-03-15 15:59:53,076: WARNING: connectionpool: Retrying (Retry(total=4, connect=5, read=4, redirect=5, status=5)) after connection broken by 'ConnectionResetError(10054, 'An existing connection was forcibly closed by the remote host', None, 10054, None)': /varun7778/Kidney-Disease-Classification-MLFlow.mlflow/api/2.0/mlflow/runs/create]
[2024-03-15 15:59:55,433: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-03-15 15:59:56,059: INFO: builder_impl: Assets written to: C:\Users\svaru\AppData\Local\Temp\tmpai427apd\model\data\model\assets]
[2024-03-15 16:58:34,775: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-03-15 16:58:34,779: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-15 16:58:34,780: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-15 16:58:34,780: INFO: common: created directory at: artifacts]
[2024-03-15 16:58:34,780: INFO: common: created directory at: artifacts/data_ingestion]
[2024-03-15 16:58:34,780: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-03-15 16:58:47,662: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-03-15 16:58:47,676: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-15 16:58:47,676: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-15 16:58:47,676: INFO: common: created directory at: artifacts]
[2024-03-15 16:58:47,676: INFO: common: created directory at: artifacts/data_ingestion]
[2024-03-15 16:58:47,676: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-03-15 16:58:59,304: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-03-15 16:58:59,877: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-03-15 16:58:59,878: INFO: main: *******************]
[2024-03-15 16:58:59,878: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-03-15 16:58:59,880: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-15 16:58:59,883: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-15 16:58:59,883: INFO: common: created directory at: artifacts]
[2024-03-15 16:58:59,884: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-03-15 16:59:00,389: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-03-15 16:59:00,605: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-03-15 16:59:00,606: INFO: main: *******************]
[2024-03-15 16:59:00,606: INFO: main: >>>>>> stage Training started <<<<<<]
[2024-03-15 16:59:00,608: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-15 16:59:00,609: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-15 16:59:00,609: INFO: common: created directory at: artifacts]
[2024-03-15 16:59:00,609: INFO: common: created directory at: artifacts\training]
[2024-03-15 17:00:15,034: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-03-15 17:00:15,037: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-15 17:00:15,037: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-15 17:00:15,037: INFO: common: created directory at: artifacts]
[2024-03-15 17:00:15,037: INFO: common: created directory at: artifacts/data_ingestion]
[2024-03-15 17:00:15,037: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-03-15 17:00:35,638: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-03-15 17:00:35,922: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-03-15 17:00:35,922: INFO: main: *******************]
[2024-03-15 17:00:35,923: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-03-15 17:00:35,924: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-15 17:00:35,925: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-15 17:00:35,925: INFO: common: created directory at: artifacts]
[2024-03-15 17:00:35,926: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-03-15 17:00:36,209: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-03-15 17:00:36,394: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-03-15 17:00:36,394: INFO: main: *******************]
[2024-03-15 17:00:36,394: INFO: main: >>>>>> stage Training started <<<<<<]
[2024-03-15 17:00:36,394: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-15 17:00:36,394: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-15 17:00:36,394: INFO: common: created directory at: artifacts]
[2024-03-15 17:00:36,394: INFO: common: created directory at: artifacts\training]
[2024-03-15 18:07:03,604: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-03-15 18:07:03,604: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-15 18:07:03,604: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-15 18:07:03,604: INFO: common: created directory at: artifacts]
[2024-03-15 18:07:03,604: INFO: common: created directory at: artifacts/data_ingestion]
[2024-03-15 18:07:03,604: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-03-15 18:07:37,211: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-03-15 18:07:37,772: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-03-15 18:07:37,773: INFO: main: *******************]
[2024-03-15 18:07:37,773: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-03-15 18:07:37,782: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-15 18:07:37,784: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-15 18:07:37,785: INFO: common: created directory at: artifacts]
[2024-03-15 18:07:37,787: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-03-15 18:07:38,339: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-03-15 18:07:38,697: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-03-15 18:07:38,698: INFO: main: *******************]
[2024-03-15 18:07:38,698: INFO: main: >>>>>> stage Training started <<<<<<]
[2024-03-15 18:07:38,700: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-15 18:07:38,702: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-15 18:07:38,702: INFO: common: created directory at: artifacts]
[2024-03-15 18:07:38,702: INFO: common: created directory at: artifacts\training]
[2024-03-15 18:11:11,810: INFO: main: >>>>>> stage Training completed <<<<<<

x==========x]
[2024-03-15 18:11:11,810: INFO: main: *******************]
[2024-03-15 18:11:11,810: INFO: main: >>>>>> stage Evaluation stage started <<<<<<]
[2024-03-15 18:11:11,812: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-15 18:11:11,813: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-15 18:11:11,813: INFO: common: created directory at: artifacts]
[2024-03-15 18:11:18,313: INFO: common: json file saved at: scores.json]
[2024-03-15 18:11:18,313: INFO: common: json file saved at: scores.json]
[2024-03-15 18:11:18,313: INFO: main: >>>>>> stage Evaluation stage completed <<<<<<

x==========x]
[2024-03-15 18:14:12,329: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-03-15 18:14:12,329: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-15 18:14:12,329: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-15 18:14:12,329: INFO: common: created directory at: artifacts]
[2024-03-15 18:14:12,329: INFO: common: created directory at: artifacts/data_ingestion]
[2024-03-15 18:14:12,329: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-03-15 18:14:23,306: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-03-15 18:14:23,592: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-03-15 18:14:23,592: INFO: main: *******************]
[2024-03-15 18:14:23,592: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-03-15 18:14:23,592: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-15 18:14:23,592: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-15 18:14:23,592: INFO: common: created directory at: artifacts]
[2024-03-15 18:14:23,592: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-03-15 18:14:23,869: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-03-15 18:14:24,040: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-03-15 18:14:24,040: INFO: main: *******************]
[2024-03-15 18:14:24,040: INFO: main: >>>>>> stage Training started <<<<<<]
[2024-03-15 18:14:24,040: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-15 18:14:24,040: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-15 18:14:24,040: INFO: common: created directory at: artifacts]
[2024-03-15 18:14:24,040: INFO: common: created directory at: artifacts\training]
[2024-03-15 18:19:46,965: INFO: main: >>>>>> stage Training completed <<<<<<

x==========x]
[2024-03-15 18:19:46,965: INFO: main: *******************]
[2024-03-15 18:19:46,965: INFO: main: >>>>>> stage Evaluation stage started <<<<<<]
[2024-03-15 18:19:46,965: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-15 18:19:46,968: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-15 18:19:46,968: INFO: common: created directory at: artifacts]
[2024-03-15 18:19:53,498: INFO: common: json file saved at: scores.json]
[2024-03-15 18:19:53,498: INFO: common: json file saved at: scores.json]
[2024-03-15 18:19:53,498: INFO: main: >>>>>> stage Evaluation stage completed <<<<<<

x==========x]
[2024-03-15 18:26:16,040: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-03-15 18:26:16,048: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-15 18:26:16,049: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-15 18:26:16,049: INFO: common: created directory at: artifacts]
[2024-03-15 18:26:16,054: INFO: common: created directory at: artifacts/data_ingestion]
[2024-03-15 18:26:16,054: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-03-15 18:26:23,034: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-03-15 18:26:23,321: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-03-15 18:26:23,321: INFO: main: *******************]
[2024-03-15 18:26:23,321: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-03-15 18:26:23,321: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-15 18:26:23,321: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-15 18:26:23,321: INFO: common: created directory at: artifacts]
[2024-03-15 18:26:23,321: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-03-15 18:26:23,589: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-03-15 18:26:23,763: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-03-15 18:26:23,763: INFO: main: *******************]
[2024-03-15 18:26:23,763: INFO: main: >>>>>> stage Training started <<<<<<]
[2024-03-15 18:26:23,767: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-15 18:26:23,767: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-15 18:26:23,767: INFO: common: created directory at: artifacts]
[2024-03-15 18:26:23,767: INFO: common: created directory at: artifacts\training]
[2024-03-15 18:27:05,949: INFO: main: >>>>>> stage Training completed <<<<<<

x==========x]
[2024-03-15 18:27:05,949: INFO: main: *******************]
[2024-03-15 18:27:05,949: INFO: main: >>>>>> stage Evaluation stage started <<<<<<]
[2024-03-15 18:27:05,957: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-15 18:27:05,958: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-15 18:27:05,958: INFO: common: created directory at: artifacts]
[2024-03-15 18:27:12,318: INFO: common: json file saved at: scores.json]
[2024-03-15 18:27:12,318: INFO: common: json file saved at: scores.json]
[2024-03-15 18:27:12,318: INFO: main: >>>>>> stage Evaluation stage completed <<<<<<

x==========x]
[2024-03-15 18:29:19,717: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-03-15 18:29:19,717: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-15 18:29:19,717: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-15 18:29:19,717: INFO: common: created directory at: artifacts]
[2024-03-15 18:29:19,717: INFO: common: created directory at: artifacts/data_ingestion]
[2024-03-15 18:29:19,717: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-03-15 18:29:29,034: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-03-15 18:29:29,321: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-03-15 18:29:29,321: INFO: main: *******************]
[2024-03-15 18:29:29,321: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-03-15 18:29:29,323: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-15 18:29:29,323: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-15 18:29:29,323: INFO: common: created directory at: artifacts]
[2024-03-15 18:29:29,323: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-03-15 18:29:29,568: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-03-15 18:29:29,743: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-03-15 18:29:29,743: INFO: main: *******************]
[2024-03-15 18:29:29,744: INFO: main: >>>>>> stage Training started <<<<<<]
[2024-03-15 18:29:29,745: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-15 18:29:29,746: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-15 18:29:29,746: INFO: common: created directory at: artifacts]
[2024-03-15 18:29:29,746: INFO: common: created directory at: artifacts\training]
[2024-03-15 18:30:09,459: INFO: main: >>>>>> stage Training completed <<<<<<

x==========x]
[2024-03-15 18:30:09,459: INFO: main: *******************]
[2024-03-15 18:30:09,459: INFO: main: >>>>>> stage Evaluation stage started <<<<<<]
[2024-03-15 18:30:09,459: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-15 18:30:09,459: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-15 18:30:09,459: INFO: common: created directory at: artifacts]
[2024-03-15 18:30:16,267: INFO: common: json file saved at: scores.json]
[2024-03-15 18:30:16,267: INFO: common: json file saved at: scores.json]
[2024-03-15 18:30:17,602: ERROR: main: INVALID_PARAMETER_VALUE: Response: {'error_code': 'INVALID_PARAMETER_VALUE'}]
Traceback (most recent call last):
  File "D:\Projects\Kidney-Disease-Classification-MLFlow\main.py", line 49, in <module>
    model_evalution.main()
  File "d:\projects\kidney-disease-classification-mlflow\src\cnnClassifier\pipeline\stage_04_model_evaluation.py", line 20, in main
    evaluation.log_into_mlflow()
  File "d:\projects\kidney-disease-classification-mlflow\src\cnnClassifier\components\model_evaluation_mlflow.py", line 60, in log_into_mlflow
    with mlflow.start_run():
  File "C:\Users\svaru\anaconda3\envs\mlflow\lib\site-packages\mlflow\tracking\fluent.py", line 350, in start_run
    active_run_obj = client.create_run(
  File "C:\Users\svaru\anaconda3\envs\mlflow\lib\site-packages\mlflow\tracking\client.py", line 275, in create_run
    return self._tracking_client.create_run(experiment_id, start_time, tags, run_name)
  File "C:\Users\svaru\anaconda3\envs\mlflow\lib\site-packages\mlflow\tracking\_tracking_service\client.py", line 131, in create_run
    return self.store.create_run(
  File "C:\Users\svaru\anaconda3\envs\mlflow\lib\site-packages\mlflow\store\tracking\rest_store.py", line 175, in create_run
    response_proto = self._call_endpoint(CreateRun, req_body)
  File "C:\Users\svaru\anaconda3\envs\mlflow\lib\site-packages\mlflow\store\tracking\rest_store.py", line 56, in _call_endpoint
    return call_endpoint(self.get_host_creds(), endpoint, method, json_body, response_proto)
  File "C:\Users\svaru\anaconda3\envs\mlflow\lib\site-packages\mlflow\utils\rest_utils.py", line 281, in call_endpoint
    response = verify_rest_response(response, endpoint)
  File "C:\Users\svaru\anaconda3\envs\mlflow\lib\site-packages\mlflow\utils\rest_utils.py", line 207, in verify_rest_response
    raise RestException(json.loads(response.text))
mlflow.exceptions.RestException: INVALID_PARAMETER_VALUE: Response: {'error_code': 'INVALID_PARAMETER_VALUE'}
[2024-03-15 18:37:39,581: INFO: main: *******************]
[2024-03-15 18:37:39,581: INFO: main: >>>>>> stage Evaluation stage started <<<<<<]
[2024-03-15 18:37:39,581: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-15 18:37:39,581: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-15 18:37:39,581: INFO: common: created directory at: artifacts]
[2024-03-15 18:37:39,955: ERROR: main: name 'model' is not defined]
Traceback (most recent call last):
  File "D:\Projects\Kidney-Disease-Classification-MLFlow\main.py", line 49, in <module>
    model_evalution.main()
  File "d:\projects\kidney-disease-classification-mlflow\src\cnnClassifier\pipeline\stage_04_model_evaluation.py", line 18, in main
    evaluation.evaluation()
  File "d:\projects\kidney-disease-classification-mlflow\src\cnnClassifier\components\model_evaluation_mlflow.py", line 48, in evaluation
    self.score = model.evaluate(self.valid_generator)
NameError: name 'model' is not defined
[2024-03-15 18:39:22,483: INFO: main: *******************]
[2024-03-15 18:39:22,483: INFO: main: >>>>>> stage Evaluation stage started <<<<<<]
[2024-03-15 18:39:22,490: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-15 18:39:22,490: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-15 18:39:22,490: INFO: common: created directory at: artifacts]
[2024-03-15 18:39:29,287: INFO: common: json file saved at: scores.json]
[2024-03-15 18:39:29,287: INFO: common: json file saved at: scores.json]
[2024-03-15 18:39:30,442: ERROR: main: INVALID_PARAMETER_VALUE: Response: {'error_code': 'INVALID_PARAMETER_VALUE'}]
Traceback (most recent call last):
  File "D:\Projects\Kidney-Disease-Classification-MLFlow\main.py", line 49, in <module>
    model_evalution.main()
  File "d:\projects\kidney-disease-classification-mlflow\src\cnnClassifier\pipeline\stage_04_model_evaluation.py", line 20, in main
    evaluation.log_into_mlflow()
  File "d:\projects\kidney-disease-classification-mlflow\src\cnnClassifier\components\model_evaluation_mlflow.py", line 60, in log_into_mlflow
    with mlflow.start_run():
  File "C:\Users\svaru\anaconda3\envs\mlflow\lib\site-packages\mlflow\tracking\fluent.py", line 350, in start_run
    active_run_obj = client.create_run(
  File "C:\Users\svaru\anaconda3\envs\mlflow\lib\site-packages\mlflow\tracking\client.py", line 275, in create_run
    return self._tracking_client.create_run(experiment_id, start_time, tags, run_name)
  File "C:\Users\svaru\anaconda3\envs\mlflow\lib\site-packages\mlflow\tracking\_tracking_service\client.py", line 131, in create_run
    return self.store.create_run(
  File "C:\Users\svaru\anaconda3\envs\mlflow\lib\site-packages\mlflow\store\tracking\rest_store.py", line 175, in create_run
    response_proto = self._call_endpoint(CreateRun, req_body)
  File "C:\Users\svaru\anaconda3\envs\mlflow\lib\site-packages\mlflow\store\tracking\rest_store.py", line 56, in _call_endpoint
    return call_endpoint(self.get_host_creds(), endpoint, method, json_body, response_proto)
  File "C:\Users\svaru\anaconda3\envs\mlflow\lib\site-packages\mlflow\utils\rest_utils.py", line 281, in call_endpoint
    response = verify_rest_response(response, endpoint)
  File "C:\Users\svaru\anaconda3\envs\mlflow\lib\site-packages\mlflow\utils\rest_utils.py", line 207, in verify_rest_response
    raise RestException(json.loads(response.text))
mlflow.exceptions.RestException: INVALID_PARAMETER_VALUE: Response: {'error_code': 'INVALID_PARAMETER_VALUE'}
[2024-03-15 18:42:18,107: INFO: main: *******************]
[2024-03-15 18:42:18,107: INFO: main: >>>>>> stage Evaluation stage started <<<<<<]
[2024-03-15 18:42:18,107: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-15 18:42:18,107: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-15 18:42:18,107: INFO: common: created directory at: artifacts]
[2024-03-15 18:42:24,778: INFO: common: json file saved at: scores.json]
[2024-03-15 18:42:24,778: INFO: common: json file saved at: scores.json]
[2024-03-15 18:42:26,011: ERROR: main: INVALID_PARAMETER_VALUE: Response: {'error_code': 'INVALID_PARAMETER_VALUE'}]
Traceback (most recent call last):
  File "D:\Projects\Kidney-Disease-Classification-MLFlow\main.py", line 49, in <module>
    model_evalution.main()
  File "d:\projects\kidney-disease-classification-mlflow\src\cnnClassifier\pipeline\stage_04_model_evaluation.py", line 20, in main
    evaluation.log_into_mlflow()
  File "d:\projects\kidney-disease-classification-mlflow\src\cnnClassifier\components\model_evaluation_mlflow.py", line 62, in log_into_mlflow
    with mlflow.start_run():
  File "C:\Users\svaru\anaconda3\envs\mlflow\lib\site-packages\mlflow\tracking\fluent.py", line 350, in start_run
    active_run_obj = client.create_run(
  File "C:\Users\svaru\anaconda3\envs\mlflow\lib\site-packages\mlflow\tracking\client.py", line 275, in create_run
    return self._tracking_client.create_run(experiment_id, start_time, tags, run_name)
  File "C:\Users\svaru\anaconda3\envs\mlflow\lib\site-packages\mlflow\tracking\_tracking_service\client.py", line 131, in create_run
    return self.store.create_run(
  File "C:\Users\svaru\anaconda3\envs\mlflow\lib\site-packages\mlflow\store\tracking\rest_store.py", line 175, in create_run
    response_proto = self._call_endpoint(CreateRun, req_body)
  File "C:\Users\svaru\anaconda3\envs\mlflow\lib\site-packages\mlflow\store\tracking\rest_store.py", line 56, in _call_endpoint
    return call_endpoint(self.get_host_creds(), endpoint, method, json_body, response_proto)
  File "C:\Users\svaru\anaconda3\envs\mlflow\lib\site-packages\mlflow\utils\rest_utils.py", line 281, in call_endpoint
    response = verify_rest_response(response, endpoint)
  File "C:\Users\svaru\anaconda3\envs\mlflow\lib\site-packages\mlflow\utils\rest_utils.py", line 207, in verify_rest_response
    raise RestException(json.loads(response.text))
mlflow.exceptions.RestException: INVALID_PARAMETER_VALUE: Response: {'error_code': 'INVALID_PARAMETER_VALUE'}
[2024-03-15 23:30:17,173: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-15 23:30:17,173: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-15 23:30:17,185: INFO: common: created directory at: artifacts]
[2024-03-15 23:30:28,463: INFO: common: json file saved at: scores.json]
[2024-03-15 23:30:28,693: WARNING: connectionpool: Retrying (Retry(total=4, connect=5, read=4, redirect=5, status=5)) after connection broken by 'ConnectionResetError(10054, 'An existing connection was forcibly closed by the remote host', None, 10054, None)': /varun7778/Kidney-Disease-Classification-MLFlow.mlflow/api/2.0/mlflow/runs/create]
[2024-03-15 23:35:04,873: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-15 23:35:04,878: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-15 23:35:04,878: INFO: common: created directory at: artifacts]
[2024-03-15 23:35:15,681: INFO: common: json file saved at: scores.json]
[2024-03-15 23:36:24,820: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-15 23:36:24,824: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-15 23:36:24,825: INFO: common: created directory at: artifacts]
[2024-03-15 23:36:35,439: INFO: common: json file saved at: scores.json]
[2024-03-15 23:37:13,606: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-15 23:37:13,609: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-15 23:37:13,609: INFO: common: created directory at: artifacts]
[2024-03-15 23:37:24,497: INFO: common: json file saved at: scores.json]
[2024-03-15 23:37:27,623: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-03-15 23:37:28,328: INFO: builder_impl: Assets written to: C:\Users\svaru\AppData\Local\Temp\tmpcbdwwm7c\model\data\model\assets]
[2024-03-15 23:41:03,214: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-15 23:41:03,221: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-15 23:41:03,221: INFO: common: created directory at: artifacts]
[2024-03-15 23:41:14,111: INFO: common: json file saved at: scores.json]
[2024-03-15 23:42:33,220: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-15 23:42:33,220: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-15 23:42:33,235: INFO: common: created directory at: artifacts]
[2024-03-15 23:42:43,400: INFO: common: json file saved at: scores.json]
[2024-03-15 23:44:41,718: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-15 23:44:41,722: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-15 23:44:41,722: INFO: common: created directory at: artifacts]
[2024-03-15 23:44:52,069: INFO: common: json file saved at: scores.json]
[2024-03-15 23:44:53,915: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-03-15 23:44:54,700: INFO: builder_impl: Assets written to: C:\Users\svaru\AppData\Local\Temp\tmpoisoulcz\model\data\model\assets]
[2024-03-15 23:48:51,315: INFO: main: *******************]
[2024-03-15 23:48:51,315: INFO: main: >>>>>> stage Evaluation stage started <<<<<<]
[2024-03-15 23:48:51,323: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-15 23:48:51,323: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-15 23:48:51,323: INFO: common: created directory at: artifacts]
[2024-03-15 23:49:02,068: INFO: common: json file saved at: scores.json]
[2024-03-15 23:49:02,068: INFO: common: json file saved at: scores.json]
[2024-03-15 23:49:05,157: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-03-15 23:49:06,009: INFO: builder_impl: Assets written to: C:\Users\svaru\AppData\Local\Temp\tmpt2nwmtg6\model\data\model\assets]
[2024-03-15 23:51:20,130: INFO: main: >>>>>> stage Evaluation stage completed <<<<<<

x==========x]
[2024-03-16 00:12:30,481: INFO: stage_01_data_ingestion: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-03-16 00:12:30,481: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-16 00:12:30,489: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-16 00:12:30,489: INFO: common: created directory at: artifacts]
[2024-03-16 00:12:30,489: INFO: common: created directory at: artifacts/data_ingestion]
[2024-03-16 00:12:30,489: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-03-16 00:13:27,604: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-03-16 00:13:28,490: INFO: stage_01_data_ingestion: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-03-16 00:13:41,944: INFO: stage_02_prepare_base_model: *******************]
[2024-03-16 00:13:41,944: INFO: stage_02_prepare_base_model: >>>>>> stage Prepare base model started <<<<<<]
[2024-03-16 00:13:41,951: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-16 00:13:41,952: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-16 00:13:41,954: INFO: common: created directory at: artifacts]
[2024-03-16 00:13:41,954: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-03-16 00:13:42,557: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-03-16 00:13:42,914: INFO: stage_02_prepare_base_model: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-03-16 00:13:50,887: INFO: stage_03_model_training: *******************]
[2024-03-16 00:13:50,888: INFO: stage_03_model_training: >>>>>> stage Training started <<<<<<]
[2024-03-16 00:13:50,888: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-16 00:13:50,888: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-16 00:13:50,894: INFO: common: created directory at: artifacts]
[2024-03-16 00:13:50,894: INFO: common: created directory at: artifacts\training]
[2024-03-16 00:15:07,534: INFO: stage_03_model_training: >>>>>> stage Training completed <<<<<<

x==========x]
[2024-03-16 00:15:16,894: INFO: stage_04_model_evaluation: *******************]
[2024-03-16 00:15:16,894: INFO: stage_04_model_evaluation: >>>>>> stage Evaluation stage started <<<<<<]
[2024-03-16 00:15:16,894: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-16 00:15:16,900: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-16 00:15:16,900: INFO: common: created directory at: artifacts]
[2024-03-16 00:15:29,403: INFO: common: json file saved at: scores.json]
[2024-03-16 00:15:29,403: INFO: common: json file saved at: scores.json]
[2024-03-16 00:15:32,470: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-03-16 00:15:33,194: INFO: builder_impl: Assets written to: C:\Users\svaru\AppData\Local\Temp\tmp2bcrc_3w\model\data\model\assets]
[2024-03-16 00:15:55,116: WARNING: connectionpool: Retrying (Retry(total=4, connect=5, read=4, redirect=5, status=5)) after connection broken by 'ConnectionAbortedError(10053, 'An established connection was aborted by the software in your host machine', None, 10053, None)': /varun7778/Kidney-Disease-Classification-MLFlow.mlflow/api/2.0/mlflow-artifacts/artifacts/7b3e6b53b4884cf1b5188d26addf17d7/f6f1b795550b45d4b6b725f225f0a3a5/artifacts/model/data/model/variables/variables.data-00000-of-00001]
[2024-03-16 00:16:37,770: INFO: stage_04_model_evaluation: >>>>>> stage Evaluation stage completed <<<<<<

x==========x]
[2024-03-16 00:41:54,302: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://10.0.0.54:8080]
[2024-03-16 00:41:54,303: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-03-16 00:42:13,301: INFO: _internal: 10.0.0.54 - - [16/Mar/2024 00:42:13] "GET / HTTP/1.1" 200 -]
[2024-03-16 00:42:44,351: ERROR: app: Exception on /predict [POST]]
Traceback (most recent call last):
  File "C:\Users\svaru\anaconda3\envs\mlflow\lib\site-packages\flask\app.py", line 2190, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\svaru\anaconda3\envs\mlflow\lib\site-packages\flask\app.py", line 1486, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\svaru\anaconda3\envs\mlflow\lib\site-packages\flask_cors\extension.py", line 176, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "C:\Users\svaru\anaconda3\envs\mlflow\lib\site-packages\flask\app.py", line 1484, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\svaru\anaconda3\envs\mlflow\lib\site-packages\flask\app.py", line 1469, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "C:\Users\svaru\anaconda3\envs\mlflow\lib\site-packages\flask_cors\decorator.py", line 130, in wrapped_function
    resp = make_response(f(*args, **kwargs))
  File "D:\Projects\Kidney-Disease-Classification-MLFlow\app.py", line 44, in predictRoute
    result = clApp.classifier.predict()
  File "d:\projects\kidney-disease-classification-mlflow\src\cnnClassifier\pipeline\prediction.py", line 16, in predict
    model = load_model(os.path.join("model", "model.h5"))
  File "C:\Users\svaru\anaconda3\envs\mlflow\lib\site-packages\keras\saving\saving_api.py", line 212, in load_model
    return legacy_sm_saving_lib.load_model(
  File "C:\Users\svaru\anaconda3\envs\mlflow\lib\site-packages\keras\utils\traceback_utils.py", line 70, in error_handler
    raise e.with_traceback(filtered_tb) from None
  File "C:\Users\svaru\anaconda3\envs\mlflow\lib\site-packages\keras\saving\legacy\save.py", line 230, in load_model
    raise IOError(
OSError: No file or directory found at model\model.h5
[2024-03-16 00:42:44,382: INFO: _internal: 10.0.0.54 - - [16/Mar/2024 00:42:44] "[35m[1mPOST /predict HTTP/1.1[0m" 500 -]
[2024-03-16 00:44:08,624: INFO: main: *******************]
[2024-03-16 00:44:08,624: INFO: main: >>>>>> stage Evaluation stage started <<<<<<]
[2024-03-16 00:44:08,624: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-16 00:44:08,624: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-16 00:44:08,624: INFO: common: created directory at: artifacts]
[2024-03-16 00:44:19,105: INFO: common: json file saved at: scores.json]
[2024-03-16 00:44:19,105: INFO: common: json file saved at: scores.json]
[2024-03-16 00:44:21,947: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-03-16 00:44:22,981: INFO: builder_impl: Assets written to: C:\Users\svaru\AppData\Local\Temp\tmpityp2jtk\model\data\model\assets]
[2024-03-16 00:45:29,518: INFO: main: >>>>>> stage Evaluation stage completed <<<<<<

x==========x]
[2024-03-16 00:46:55,418: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://10.0.0.54:8080]
[2024-03-16 00:46:55,418: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-03-16 00:47:01,124: INFO: _internal: 127.0.0.1 - - [16/Mar/2024 00:47:01] "GET / HTTP/1.1" 200 -]
[2024-03-16 00:47:13,337: INFO: _internal: 127.0.0.1 - - [16/Mar/2024 00:47:13] "POST /predict HTTP/1.1" 200 -]
[2024-03-16 00:47:29,825: INFO: _internal: 127.0.0.1 - - [16/Mar/2024 00:47:29] "POST /predict HTTP/1.1" 200 -]
[2024-03-16 00:47:32,458: INFO: _internal: 127.0.0.1 - - [16/Mar/2024 00:47:32] "POST /predict HTTP/1.1" 200 -]
[2024-03-16 00:47:34,158: INFO: _internal: 127.0.0.1 - - [16/Mar/2024 00:47:34] "POST /predict HTTP/1.1" 200 -]
[2024-03-16 00:47:36,821: WARNING: polymorphic_function: 5 out of the last 5 calls to <function Model.make_predict_function.<locals>.predict_function at 0x00000218F23D55E0> triggered tf.function retracing. Tracing is expensive and the excessive number of tracings could be due to (1) creating @tf.function repeatedly in a loop, (2) passing tensors with different shapes, (3) passing Python objects instead of tensors. For (1), please define your @tf.function outside of the loop. For (2), @tf.function has reduce_retracing=True option that can avoid unnecessary retracing. For (3), please refer to https://www.tensorflow.org/guide/function#controlling_retracing and https://www.tensorflow.org/api_docs/python/tf/function for  more details.]
[2024-03-16 00:47:36,825: INFO: _internal: 127.0.0.1 - - [16/Mar/2024 00:47:36] "POST /predict HTTP/1.1" 200 -]
[2024-03-16 00:49:13,269: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://10.0.0.54:8080]
[2024-03-16 00:49:13,269: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-03-16 00:49:26,491: INFO: _internal: 127.0.0.1 - - [16/Mar/2024 00:49:26] "GET / HTTP/1.1" 200 -]
[2024-03-16 00:49:38,766: INFO: _internal: 127.0.0.1 - - [16/Mar/2024 00:49:38] "POST /predict HTTP/1.1" 200 -]
[2024-03-16 00:49:52,455: INFO: _internal: 127.0.0.1 - - [16/Mar/2024 00:49:52] "POST /predict HTTP/1.1" 200 -]
